src/libraries/System.Threading.Channels/tests/ChannelTestBase.cs

```diff
@@ -892,7 +892,7 @@ public async Task WriteAsync_AwaitThenGetResult_Throws()
             Assert.Throws<InvalidOperationException>(() => write.GetAwaiter().GetResult());
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
Same question: what about this test requires this suppression?

Reply:
The test was fine, I was mistaken about the hang.